Source/Libraries/NetCore/NetCoreSpec.cs

```diff
@@ -9,18 +9,17 @@ public class NetCoreSpec : IDisposable
         // This is a parameters object that must be passed to netcore in order to establish a link
         // The values here will determine the behavior of the Link
 
-        public NetCoreConnector Connector = null;
-        public NetworkSide Side = NetworkSide.NONE;
-        public bool AutoReconnect = true;
-        public int ClientReconnectDelay = 1500;
-        public int DefaultBoopMonitoringCounter = System.Diagnostics.Debugger.IsAttached ? 1500 : 20;
-
-        public bool Attached = true;
-        public bool Loopback = true;
-        public string IP = "127.0.0.1";
-        public int Port = 42069;
-
-        public int messageReadTimerDelay = 5; //represents how often the messages are read (ms) (15ms = ~66fps)
+        public NetCoreConnector Connector { get; set; } = null;
+        public NetworkSide Side { get; set; } = NetworkSide.NONE;
+        public int ClientReconnectDelay { get; set; } = 1500;
+        public int DefaultBoopMonitoringCounter { get; } = System.Diagnostics.Debugger.IsAttached ? 1500 : 20;
+
+        public bool Attached { get; set; } = true;
```

Comment:
In the future, we should move these over to a constructor and have them all get; because there's no reason for them to be set after being initialized. That'll cause a lot of implementation breakage though so for now, we can do this.

We should start an ongoing list of "things to change that'll require a rebuild of all implementations" and a list of "things to change when breaking compatibility with existing stockpiles. Might be worth enabling issues on the repo and using an issue to track progress.

Reply:
+1 for using Repo issues for this (the new GitHub org should have some nice features too).

We can use various tags for "Breaks existing stockpiles" and "breaks existing implementations"